tests/unit/test_specs.py

```diff
@@ -90,6 +107,12 @@ def test_get_product_bundles():
     assert TEST_PRODUCT_BUNDLE in bundles
 
 
-def test_get_item_types():
+def test_get_item_types_with_bundle():
     item_types = specs.get_item_types(TEST_PRODUCT_BUNDLE)
```

Comment:
Let's use the keyword argument form

```suggestion
    item_types = specs.get_item_types(product_bundle=TEST_PRODUCT_BUNDLE)
```